### ISSUE TITLE
removeEnumPrefix generating wrong code in callsites

### ISSUE THREAD

**Issue by anzorbalkar** [stephenh/ts-proto#830]
Hi,

I just started using the `removeEnumPrefix` flag and it seems to be working well to generate the actual enum, however the other callsites in the generated code are still referencing the old style naming. In other words the generated code itself is inconsistent.

Example:

```
// product.proto

enum Color {
  COLOR_UNSPECIFIED = 0;
  COLOR_RED = 1;
  COLOR_GREEN = 2;
  COLOR_BLUE = 3;
}

message Product {
  Color color = 1;
}
```

=>

```
// product.ts

// good; expected.
export enum Color {
  UNSPECIFIED = "COLOR_UNSPECIFIED",
  RED = "COLOR_RED",
  GREEN = "COLOR_GREEN",
  BLUE = "COLOR_BLUE",
  UNRECOGNIZED = "UNRECOGNIZED",
}

...

function createBaseProduct(): Product {
  return { color: Color.COLOR_UNSPECIFIED };  // <-- not good; not expected; does not compile.
}
```

Am I needing to add an additional flag to correct this or is this a bug?

**Comment by stephenh** [comment]
Ah shoot, sorry for just getting to this, but yeah @anzorbalkar that is odd; I'm surprised no one else has gotten tripped up by this/that the original PR missed it.

If you could submit a PR to fix, that would be great! Also cc @superclaw who submitted the original `removeEnumPrefix` support, in case they can help out; thanks!

**Comment by superclaw** [comment]
@anzorbalkar Could you please make a repo with minimal reproduction of this bug? Do you mean generating services from proto?

**Comment by buzap** [comment]
I believe this is the same issue as #897, which was fixed in PR #902.

**Comment by stephenh** [comment]
Ah yep! Agreed @buzap , thanks for noticing that! Closing this out.
